packages/components/src/DataList/DataList.types.ts

```diff
@@ -44,6 +44,11 @@ export type DataListHeader<T extends DataListObject> = {
   readonly [K in keyof T]?: string;
 };
 
+export interface DataListSorting {
+  readonly key: string;
+  readonly direction: "asc" | "desc" | "none";
```

Comment:
what's the `none` for?

Reply:
For when the consumer wants to reset the sorting direction, but now thinking better, just setting the key to an empty string should work too